### ISSUE TITLE
Can you write a complete example

### ISSUE THREAD

**Issue by codexuexi** [nicksnyder/go-i18n#220]
```go
	Description string

	// LeftDelim is the left Go template delimiter.
	LeftDelim string

	// RightDelim is the right Go template delimiter.``
	RightDelim string

	// Zero is the content of the message for the CLDR plural form "zero".
	Zero string

	// One is the content of the message for the CLDR plural form "one".
	One string

	// Two is the content of the message for the CLDR plural form "two".
	Two string

	// Few is the content of the message for the CLDR plural form "few".
	Few string

	// Many is the content of the message for the CLDR plural form "many".
	Many string

	// Other is the content of the message for the CLDR plural form "other".
	Other string
```
```go
package main

import (
	"fmt"
	"github.com/BurntSushi/toml"
	"golang.org/x/text/language"
)
import "github.com/nicksnyder/go-i18n/v2/i18n"

func main() {
	bundle := i18n.NewBundle(language.English)
	bundle.RegisterUnmarshalFunc("toml", toml.Unmarshal)
	bundle.LoadMessageFile("es.toml")
	lang := "es"
	accept := ""
	localizer := i18n.NewLocalizer(bundle, lang, accept)

	localize, err := localizer.Localize(&i18n.LocalizeConfig{
		DefaultMessage: &i18n.Message{
			ID:    "PersonCats",
			Zero:"一个单",
			One:   "{{.Name}} has {{.Count}} cat.",
			Few:   "3",
			Other: "{{.Name}} has {{.Count}} cats.",
		},
		TemplateData: map[string]interface{}{
			"Name":  "Nick",
			//"Count": 2,
		},
		PluralCount: 0,
	})
	fmt.Println(localize, err )
}
```
I don't know how to use

`One` `Two` how to use

**Comment by CarsonSlovoka** [comment]
It's a good habit:  **share** knowledge.

If you already know the answer. Please share, not close the issue directly.

----

TLDR 
> Message should correspond to the language

I mean your language is "en" or "es", unfortunately, they do not support ``One`` and ``Two``

see source code: 

> https://github.com/nicksnyder/go-i18n/blob/e2aedcfb27c41ac8dacb3e09868b18eeb5d4bc57/v2/internal/plural/rule_gen.go#L46-L47

only support [One], [Other]

----

You can change to [Arabic] since it supports varies a lot.

> 📙 Arabic is ``ar``

>https://github.com/nicksnyder/go-i18n/blob/e2aedcfb27c41ac8dacb3e09868b18eeb5d4bc57/v2/internal/plural/rule_gen.go#L535-L559

[One]: https://github.com/nicksnyder/go-i18n/blob/0c6ce6ac1e8c119993c3d5638af4f7cd67ff068a/v2/internal/plural/form.go#L11
[Other]: https://github.com/nicksnyder/go-i18n/blob/0c6ce6ac1e8c119993c3d5638af4f7cd67ff068a/v2/internal/plural/form.go#L15

[Arabic]: https://github.com/golang/text/blob/3115f89c4b99a620c7f1a4395a2b4405e95b82b6/language/tags.go#L68


## full example

```go
package main

import (
	"fmt"
	"github.com/BurntSushi/toml"
	"golang.org/x/text/language"
)
import "github.com/nicksnyder/go-i18n/v2/i18n"

func main() {
	bundle := i18n.NewBundle(language.Arabic)
	bundle.RegisterUnmarshalFunc("toml", toml.Unmarshal)
	localizer := i18n.NewLocalizer(bundle, "ar")

	for _, curCount := range []interface{}{0, 1, 2,
		3, 10, // Few range
		11, 99, // many
		nil, // other
	} {
		localize := localizer.MustLocalize(&i18n.LocalizeConfig{
			DefaultMessage: &i18n.Message{
				ID:    "PersonCats",
				Zero:  "zero 0",
				One:   "1 一",
				Two:   "2 二",
				Few:   "3-10",
				Many:  "11-99",
				Other: "{{.Name}} has {{.Count}} cats.",
			},
			TemplateData: map[string]interface{}{
				"Name":  "Nick",
				"Count": 2,
			},
			PluralCount: curCount,
		})
		fmt.Println(localize)
	}

}
```

output
```
zero 0
1 一
2 二
3-10
3-10
11-99
11-99
Nick has 2 cats.
```

**Comment by CarsonSlovoka** [comment]
related issue

> https://github.com/nicksnyder/go-i18n/issues/232#issuecomment-709542498

**Comment by CarsonSlovoka** [comment]
You may want to know how to know language code? <sup>for example ``ar = Arabic``<sup>

The below article may help you.

- [iso639-2](https://www.loc.gov/standards/iso639-2/php/code_list.php)
- [iso639-3](https://iso639-3.sil.org/code)
